<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>2. Species Richness and Diversity</title>
    <meta charset="utf-8" />
    <meta name="author" content="Jasper Slingsby, BIO3018F" />
    <meta name="date" content="2022-01-18" />
    <script src="2_SpeciesRichAndDiv_files/header-attrs/header-attrs.js"></script>
    <script src="2_SpeciesRichAndDiv_files/kePrint/kePrint.js"></script>
    <link href="2_SpeciesRichAndDiv_files/lightable/lightable.css" rel="stylesheet" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# 2. Species Richness and Diversity
## Alpha Diversity
### Jasper Slingsby, BIO3018F
### 2022-01-18

---






layout: false

.pull-left[
## The components of diversity

Alpha `\((\alpha)\)` = mean species diversity/richness within local-scale sites, habitats or communities (often termed "point diversity")

Beta `\((\beta)\)` = diversity/richness differences between sites or habitats (attributable to species "turnover" or "nestedness")

Gamma `\((\gamma)\)` = total landscape species diversity/richness

.footnote[Whittaker (1972)]
]

.pull-right[
&lt;img src="images/diversitycomponents.png" width="110%" style="display: block; margin: auto 0 auto auto;" /&gt;
]

---

layout: false

## Quantifying `\(\alpha\)`-diversity...

Can be counts of the number of species in samples, or include information on their relative abundance.

.pull-left[

&lt;img src="images/alphadiversity.png" width="75%" style="display: block; margin: auto;" /&gt;

]

.pull-right[

*S* = a simple count of species

*H'* = Species diversity quantified using the Shannon index `\((H' = -\sum_{i=1}^{n}(p_i*ln(p_i))\)` where `\(p_i\)` is the proportion of individuals belonging to the *i*th species. 

*H'* takes into account both species richness and evenness in their relative abundance.

A community is considered more diverse if the relative abundance of species is more evenly distributed. The logic being that if you remove one individual, more even communities are less likely to lose species.
]

---

layout: false

.pull-left[
## Quantifying `\(\alpha\)`-diversity...

Many studies will refer to species counts as "species richness", but this is often in error...

Species counts are sensitive to the number of individuals sampled. 

More individuals means greater potential to encounter more species.

&lt;img src="images/taylorplot.gif" width="60%" style="display: block; margin: auto;" /&gt;

]

.pull-right[
&lt;img src="images/spp_ind_accum_CP.png" width="100%" style="display: block; margin: auto 0 auto auto;" /&gt;

##### Species counts relative to number of individuals sampled across eighty-one 50m&lt;sup&gt;2&lt;/sup&gt; vegetation plots in Table Mountain National Park (data from Slingsby et al. 2017).

The relationship is non-linear and asymptotic.
]

---

layout: false

.pull-left[
## Quantifying `\(\alpha\)`-diversity...

Dependence of species counts on number of individuals sampled is problematic when the number of individuals varies between samples (e.g. vegetation plots, netting events, etc).

&lt;img src="images/fire_age.png" width="100%" style="display: block; margin: auto;" /&gt;

While raw species counts can still be useful for many purposes, it is also useful to have a metric that is independent of abundance and sampling effort.

]

.pull-right[
&lt;img src="images/spp_abund_CP.png" width="90%" style="display: block; margin: auto;" /&gt;

##### Species counts relative to number of individuals sampled for each of eighty-one 50m&lt;sup&gt;2&lt;/sup&gt; vegetation plots in Table Mountain National Park (data from Slingsby et al. 2017).

Since the samples are of a set unit area (50m&lt;sup&gt;2&lt;/sup&gt;), the species counts in this figure can also be termed ***species density***.
]

---

layout: false

.pull-left[
## Quantifying `\(\alpha\)`-diversity...

Dependence of species counts on number of individuals sampled is problematic when the number of individuals varies between samples (e.g. vegetation plots, netting events, etc).

&lt;img src="images/forest_fynbos.png" width="100%" style="display: block; margin: auto 0 auto auto;" /&gt;

While raw species counts can still be useful for many purposes, it is also useful to have a metric that is independent of abundance and sampling effort.

]

.pull-right[
&lt;img src="images/forestyoung.jpg" width="75%" style="display: block; margin: auto 0 auto auto;" /&gt;


&lt;img src="images/fynbos.png" width="75%" style="display: block; margin: auto 0 auto auto;" /&gt;

]

---

layout: false

.pull-left[
## Quantifying `\(\alpha\)`-diversity...

###*Species richness* 
...is usually reserved to mean an estimate of the richness of species while taking abundance and sampling effort into account (see Gotelli and Colwell 2001 for review).

...is often estimated using individual or sample-based ***rarefaction***. 

This is done by resampling a set number (*n*) of individuals or samples from the full dataset to determine the expected number of species for any given *n*. For each *n* the resampling is repeated multiple times and the expected species number determined as an average.

]

.pull-right[
&lt;img src="images/spec_ind_raref_CP.png" width="100%" style="display: block; margin: auto 0 auto auto;" /&gt;

##### Individual-based rarefaction curve developed by sampling individuals from the pooled data from eighty-one 50m&lt;sup&gt;2&lt;/sup&gt; vegetation plots in Table Mountain National Park (data from Slingsby et al. 2017).
]

---

layout: false

.pull-left[
###Individual-based *accumulation* curve 
&lt;img src="images/spp_ind_accum_CP.png" width="85%" style="display: block; margin: auto;" /&gt;

Curve is not smooth, because individuals (and thus species) are added in whatever order (in this case alphabetically by plot).

]

.pull-right[
###Individual-based *rarefaction* curve 

&lt;img src="images/spec_ind_raref_CP.png" width="85%" style="display: block; margin: auto;" /&gt;

Curve is smooth, because individuals (and thus species) are repeatedly sampled and averaged for each sample size.

]

---

layout: false

.pull-left[
###Sample-based *accumulation* curve 
&lt;img src="images/spp_sites_accum_CP.png" width="85%" style="display: block; margin: auto;" /&gt;

Curve is not smooth, because samples (and thus species) are added in whatever order (in this case by plot number).

]

.pull-right[
###Sample-based *rarefaction* curve 

&lt;br&gt;

&lt;img src="images/spec_ind_raref_CP.png" width="85%" style="display: block; margin: auto;" /&gt;

Curve is smooth, because samples (and thus species) are repeatedly sampled and averaged for each sample size.

]

---

layout: false

##Estimating asymptotic species richness by extrapolation?

---

layout: false

##Comparing diversity between sites or surveys

.pull-left[
###Sample-based *rarefaction* 
&lt;img src="images/spec_sites_raref_CP_comp.png" width="85%" style="display: block; margin: auto;" /&gt;

]

.pull-right[
###Individual-based *rarefaction* 

&lt;img src="images/spec_ind_raref_CP_comp.png" width="85%" style="display: block; margin: auto;" /&gt;
]

Here I compare surveys of permanent veg plots in the Cape of Good Hope section of TMNP from 1966 and 1996.

---

layout: false

##Comparing diversity between *samples*

.pull-left[
###Raw species counts 
&lt;img src="images/spec_comp_CP.png" width="65%" style="display: block; margin: auto;" /&gt;

]

.pull-right[
###Individual-based *rarefaction*&lt;sup&gt;1&lt;/sup&gt;

&lt;img src="images/sper_comp_CP.png" width="65%" style="display: block; margin: auto;" /&gt;

]

.footnote[&lt;sup&gt;1&lt;/sup&gt;for *n* = 10 individuals]

---

layout: false

##Comparing diversity between *samples*

.pull-left[
###Raw species counts 
&lt;img src="images/spec_comp_hist_CP.png" width="65%" style="display: block; margin: auto;" /&gt;

Paired T-test: t = 1.4957, df = 80, p-value = 0.1387
]

.pull-right[
###Individual-based *rarefaction*&lt;sup&gt;1&lt;/sup&gt;

&lt;img src="images/sper_comp_hist_CP.png" width="65%" style="display: block; margin: auto;" /&gt;

Paired T-test: t = 3.2953, df = 80, p-value = _***0.0015***_
]

---

layout: false

.pull-left[
## The decomposition of diversity...

The components of biodiversity are interrelated. How this is expressed depends on your interpretation of `\(\beta\)`-diversity, which varies the application at hand.


As "turnover" it is multiplicative (*sensu* Whittaker 1972):

`$$\beta = \gamma/\alpha$$`

As "average amount of diversity not found in a single, randomly-chosen sample" it is additive&lt;sup&gt;1&lt;/sup&gt; (*sensu* MacArthur et al. 1966, Lande 1996):

`$$\beta = \gamma - \alpha$$`

.footnote[&lt;sup&gt;1&lt;/sup&gt;often termed "additive partitioning of diversity". For reviews see Veech et al. 2002 and Anderson et al. 2011.]
]

.pull-right[
&lt;img src="images/diversitycomponents.png" width="110%" style="display: block; margin: auto 0 auto auto;" /&gt;
]

---

## References

Anderson, M. J., T. O. Crist, J. M. Chase, et al. (2011). "Navigating
the multiple meanings of beta-diversity: a roadmap for the practicing
ecologist". En. In: _Ecology letters_ 14.1, pp. 19-28. ISSN: 1461-023X,
1461-0248. DOI:
[10.1111/j.1461-0248.2010.01552.x](https://doi.org/10.1111%2Fj.1461-0248.2010.01552.x).

Gotelli, N. J. and R. K. Colwell (2001). "Quantifying biodiversity:
procedures and pitfalls in the measurement and comparison of species
richness". In: _Ecology letters_ 4.4, pp. 379-391. ISSN: 1461-023X,
1461-0248. DOI:
[10.1046/j.1461-0248.2001.00230.x](https://doi.org/10.1046%2Fj.1461-0248.2001.00230.x).

Lande, R. (1996). "Statistics and Partitioning of Species Diversity,
and Similarity among Multiple Communities". In: _Oikos_ 76.1, pp. 5-13.
ISSN: 0030-1299, 1600-0706. DOI:
[10.2307/3545743](https://doi.org/10.2307%2F3545743).

MacArthur, R., H. Recher, and M. Cody (1966). "On the Relation between
Habitat Selection and Species Diversity". In: _The American naturalist_
100.913, pp. 319-332. ISSN: 0003-0147. DOI:
[10.1086/282425](https://doi.org/10.1086%2F282425).

Slingsby, J. A., C. Merow, M. Aiello-Lammens, et al. (2017).
"Intensifying postfire weather and biological invasion drive species
loss in a Mediterranean-type biodiversity hotspot". En. In:
_Proceedings of the National Academy of Sciences of the United States
of America_ 114.18, pp. 4697-4702. ISSN: 0027-8424, 1091-6490. DOI:
[10.1073/pnas.1619014114](https://doi.org/10.1073%2Fpnas.1619014114).

Veech, J. A., K. S. Summerville, T. O. Crist, et al. (2002). "The
additive partitioning of species diversity: recent revival of an old
idea". En. In: _Oikos_ 99.1, pp. 3-9. ISSN: 0030-1299, 1600-0706. DOI:
[10.1034/j.1600-0706.2002.990101.x](https://doi.org/10.1034%2Fj.1600-0706.2002.990101.x).

Whittaker, R. H. (1972). "Evolution and measurement of species
diversity". En. In: _Taxon_ 21.2-3, pp. 213-251. ISSN: 0040-0262,
1996-8175. DOI: [10.2307/1218190](https://doi.org/10.2307%2F1218190).

---

## Understanding the role of biodiversity...

.pull-left[
### ...requires measuring it

Here we see relationships between biodiversity metrics and urban ecosystem services (UES) that have been tested empirically (from Schwarz et al. 2017, _Ecosystem Services_).
]

.pull-right[
&lt;img src="images/ecosystem_functions.jpg" width="100%" style="display: block; margin: auto 0 auto auto;" /&gt;
]

---

.pull-left[
## There are many facets of biodiversity to measure!

&gt; "biodiversity is the sum total of all biotic variation from the level of genes to ecosystems" - Purvis &amp; Hector 2000, _Nature_
]

.pull-right[
&lt;img src="images/Noss_Biodiversity.png" width="100%" style="display: block; margin: auto;" /&gt;
.footnote[Noss 1990, _Conservation Biology_]
]

---

.pull-left[
## There are many facets of biodiversity to measure!

#### Essential Biodiversity Variables (EBVs)

&lt;img src="images/ebv_circle.png" width="80%" style="display: block; margin: auto;" /&gt;

.footnote[https://geobon.org/]
]

.pull-right[
&lt;table class="table" style="font-size: 12px; margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:left;"&gt; EBV.class &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; EBV.name &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Genetic composition &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Genetic diversity (richness and heterozygosity) &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Genetic differentiation (number of genetic units and genetic distance) &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Effective population size &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Inbreeding &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Species populations &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Species distributions &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Species abundances &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Species traits &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Morphology &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Physiology &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Phenology &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Movement &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Community composition &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Community abundance &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Taxonomic/phylogenetic diversity &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Trait diversity &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Interaction diversity &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Ecosystem functioning &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Primary productivity &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Ecosystem phenology &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Ecosystem disturbances &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Ecosystem structure &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Live cover fraction &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Ecosystem distribution &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Ecosystem Vertical Profile &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
]

---

.pull-left[
## And many ways of measuring them!

#### Many kinds of diversity
*   Genetic diversity
*   Taxonomic diversity
*   Phylogenetic diversity
*   Functional diversity
*   Ecosystem diversity
*   etc...

#### And many methods for measuring each of them!!!
]

.pull-right[
&lt;img src="images/diversitymeasures.png" width="100%" style="display: block; margin: auto;" /&gt;
.footnote[A comparison of **_some_** phylogenetic diversity indices from Slingsby 2011, _PhD Thesis_]
]

---
class: center, middle

# Thanks!

Slides created via the R packages:

[**xaringan**](https://github.com/yihui/xaringan)&lt;br&gt;
[gadenbuie/xaringanthemer](https://github.com/gadenbuie/xaringanthemer)

The chakra comes from [remark.js](https://remarkjs.com), [**knitr**](http://yihui.name/knitr), and [R Markdown](https://rmarkdown.rstudio.com).
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"slideNumberFormat": "%current% / %total%",
"highlightStyle": "github",
"highlightLines": true,
"ratio": "16:9",
"countIncrementalSlides": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
